estonian_e_invoice/validation/validators.py

```diff
@@ -1,6 +1,81 @@
-from cerberus import Validator
+from datetime import datetime
+from decimal import Decimal
 
+from cerberus import TypeDefinition, Validator
+from estonian_e_invoice.entities import (
+    VAT,
+    AccountInfo,
+    BuyerParty,
+    ContactData,
+    InvoiceInformation,
+    InvoiceItemGroup,
+    InvoiceSumGroup,
+    InvoiceType,
+    ItemDetailInfo,
+    ItemEntry,
+    LegalAddress,
+    PaymentInfo,
+    SellerParty,
+)
 
-class BaseValidator(Validator):
-    def validate_object(self, obj):
-        return self.validate(obj.__dict__)
+DECIMAL_TYPE = TypeDefinition("decimal", (Decimal,), ())
```

Comment:
Possibly those could also be in separate file (either validation_schemas.py or entirely separate).